recipes/libcusparse/meta.yaml

```
@@ -0,0 +1,156 @@
+      - lib/pkgconfig                                 # [linux]
+      - targets/{{ target_name }}/include             # [linux]
+      - targets/{{ target_name }}/lib/libcusparse.so  # [linux]
+      - targets/{{ target_name }}/lib/stubs           # [linux]
+      # Not shipping Fortran bindings (in src) on Windows for now
+      - targets/{{ target_name }}/libcusparse/src     # [linux]
+      - Library\include                             # [win]
+      - Library\lib                                 # [win]
+    requirements:
+      build:
+        - {{ compiler("c") }}
+        - {{ compiler("cxx") }}
+        - arm-variant * {{ arm_variant_type }}  # [aarch64]
+        - sysroot_{{ target_platform }} 2.17    # [linux]
+      host:
+        - cuda-version {{ cuda_version }}
+      run:
+        - {{ pin_compatible("cuda-version", max_pin="x.x") }}
+        - {{ pin_subpackage("libcusparse", exact=True) }}
+      run_constrained:
+        - libcusparse-static >={{ version }}  # [linux]
+    test:
+      commands:
+        - test -L $PREFIX/lib/libcusparse.so                                                   # [linux]
+        - test -f $PREFIX/lib/pkgconfig/cusparse*.pc                                           # [linux]
+        - test -f $PREFIX/targets/{{ target_name }}/include/cusparse.h                         # [linux]
+        - test -L $PREFIX/targets/{{ target_name }}/lib/libcusparse.so                         # [linux]
+        - test -f $PREFIX/targets/{{ target_name }}/lib/stubs/libcusparse.so                   # [linux]
+        - test -f $PREFIX/targets/{{ target_name }}/libcusparse/src/cusparse_fortran.c         # [linux]
+        - test -f $PREFIX/targets/{{ target_name }}/libcusparse/src/cusparse_fortran.h         # [linux]
+        - test -f $PREFIX/targets/{{ target_name }}/libcusparse/src/cusparse_fortran_common.h  # [linux]
+        - if not exist %LIBRARY_INC%\cusparse.h exit 1                                         # [win]
+        - if not exist %LIBRARY_LIB%\cusparse.lib exit 1                                       # [win]
+    about:
+      home: https://developer.nvidia.com/cusparse
+      license_file: LICENSE
+      license: LicenseRef-NVIDIA-End-User-License-Agreement
+      license_url: https://docs.nvidia.com/cuda/eula/index.html
+      summary: CUDA Sparse Matrix Library
+      description: |
+        cuSPARSE - Basic Linear Algebra for Sparse Matrices on NVIDIA GPUs
+      doc_url: https://docs.nvidia.com/cuda/cusparse/index.html
+
+  - name: libcusparse-static
+    build:
+      skip: true  # [win]
+    files:
+      - targets/{{ target_name }}/lib/libcusparse_static.a
+    requirements:
+      build:
+        - {{ compiler("c") }}
+        - {{ compiler("cxx") }}
+        - arm-variant * {{ arm_variant_type }}  # [aarch64]
+        - sysroot_{{ target_platform }} 2.17    # [linux]
+      host:
+        - cuda-version {{ cuda_version }}
+      run:
+        - {{ pin_compatible("cuda-version", max_pin="x.x") }}
```

Add arm-variant constraint.
```suggestion
      run:
        - {{ pin_compatible("cuda-version", max_pin="x.x") }}
      run_constrained:
        - arm-variant * {{ arm_variant_type }} # [aarch64]
```

---

Good catch, thanks!